Documentation/ApiOverview/BackendModules/ToplevelModules.rst

```diff
@@ -0,0 +1,63 @@
+.. include:: /Includes.rst.txt
+.. index:: Backend modules; Toplevel
+
+.. _backend-modules-toplevel-module:
+
+================
+Toplevel modules
+================
+
+The following toplevel modules are provided by the Core:
+
+`web`: Web
+   All modules requiring a page tree by default. These modules are mostly used
+   to manage content that should be displayed in the frontend.
+
+`site`: Site Management
+   Settings for the complete site such as redirects and site settings.
+
+`file`: File
+   All modules requiring a file system tree such as modules dealing with file
+   metadata, uploading etc.
+
+`tools`: Admin Tools
+   By convention modules in this toplevel section should only be
+   available for admins with system-maintainer rights. Therefore the
+   configuration array of a module displayed here should always have the
+   following key-value pair: :php:`'access' => 'systemMaintainer'`.
+
+   In this toplevel section modules that deal with installing, updating the Core
+   and extensions. System-wide settings are also found here.
+
+`system`: System
+   By convention modules in this toplevel section should only be
+   available for admins. Therefore the configuration array of a module
+   displayed here should always have the following key-value pair:
+   :php:`'access' => 'admin'`.
+
+   In this toplevel section modules are situated that deal with backend user
+   rights or might reveal security relevant data.
```

Comment:
```suggestion
   In this toplevel section are modules that that deal with backend user
   rights and security-relevant data.
```

Reply:
Something is still odd with this sentence. that is duplicated?

Reply:
I think there was one excess that, removed it